test/Microsoft.TemplateEngine.Orchestrator.RunnableProjects.UnitTests/RunnableProjectGeneratorTests.cs

```diff
@@ -219,5 +220,213 @@ public async void CreateAsyncTest_ConditionWithUnquotedChoiceLiteral(string temp
             string resultContent = environment.Host.FileSystem.ReadAllText(Path.Combine(targetDir, "sourcFile")).Trim();
             Assert.Equal(expectedResult, resultContent);
         }
+
+        [Fact]
+        public async void CreateAsyncTest_MultiChoiceParamReplacingAndCondition()
+        {
+            //
+            // Template content preparation
+            //
+
+            string templateConfig = @"
```

Comment:
(not actionable):
couple of ideas how to avoid strings (they a hard to manage)
- use anonymous classes where members match properties
- after recent refactoring you can create `SimpleConfigModel` and init its fields

Reply:
I actually considered those test to be a typo of e2e test that can be a bit more performant - so I prefered the input to be as unprocessed as possible